src/traits.rs

```diff
@@ -198,14 +170,49 @@ pub trait Integer:
     }
 }
 
-/// Signed integers.
-pub trait Signed: Integer {
+/// Signed [`Integer`]s.
+pub trait Signed:
```

Comment:
Are there additional basic trivial things we can add here?

- [ ] `From<i8> + From<i16> + From<i32>` (I now notice `Unsigned` has `From<u64>` though that might panic on 32-bit for `Uint<1>`?)